runtime/common/src/crowdloan/migration.rs

```diff
@@ -78,10 +105,21 @@ pub mod crowdloan_index_migration {
 
 			// Funds should only have a free balance and a reserve balance. Both of these are in the
 			// `Account` storage item, so we just swap them.
-			let account_info = frame_system::Account::<T>::take(old_fund_account);
-			frame_system::Account::<T>::insert(new_fund_account, account_info);
+			let account_info = frame_system::Account::<T>::take(&old_fund_account);
+			frame_system::Account::<T>::insert(&new_fund_account, account_info);
```

Comment:
Just to make sure I understand, if this migration runs twice things break; we would overwrite `new_fund_account` here with some empty info

Reply:
Yes, but not sure if there is a way around this...

Reply:
you can and should guard the migration by something, either `StorageVersion`, custom storage, or at the minimum `spec_verion`